fastapi_events/dispatcher.py

```diff
@@ -61,12 +63,22 @@ def _dispatch(event_name: Union[str, Enum], payload: Optional[Any] = None) -> No
         _dispatch_as_task(event_name, payload)
 
 
+@contextlib.contextmanager
+def set_middleware_identifier(middleware_id: int) -> Iterator[None]:
```

Comment:
Since `set_middleware_identifier` is not expected to be used outside of this module, what about prefixing an underscore `_` to the function? (-> `_set_middleware_identifier`)